lib/integrations/awsoidc/listdatabases.go

```diff
@@ -188,7 +192,15 @@ func listDBClusters(ctx context.Context, clt ListDatabasesClient, req ListDataba
 			continue
 		}
 
-		awsDB, err := services.NewDatabaseFromRDSV2Cluster(&db)
+		// RDS Clusters do not return VPC and Subnets.
+		// To get this value, a member of the cluster is fetched and its Network Information is used to
+		// populate the RDS Cluster information.
```

Comment:
Is there a chance that instances could have different network information? Should we mention that here?

Reply:
I'll add the following text
> 		// All the members have the same network information, so picking one at random should not matter.

Reply:
That's good, thanks.